src/keboola/component/dao.py

```diff
@@ -1251,8 +1251,7 @@ def columns(self, val: List[str]):
                 raise TypeError("Columns must be a list")
 
             for col in val:
-                if col not in self.columns:
-                    self.schema[col] = ColumnDefinition()
+                self.schema[col] = ColumnDefinition()
```

Comment:
Promin ale tohle porad neni to co jsem myslel https://github.com/keboola/python-component/pull/55/commits/4f3c6dd6cb0cdd45ee42a8f219ee95ed20648e71#diff-6615ea46ac4ee88c1db70b9f0623e257843f25b60b46d65de76a2c8fdbac64bcR1246 tady to potichu zarve a uzivatel si bude myslet ze se neco stalo, ale nic se nestalo. Zachoval bych tu puvodni funkcionalitu a tady prepsal cely schema a vyhodil ten check

Reply:
změněno, takže uživatel si může přepsat celé schema jen nastavením columns a jenom to vypíše warning?